src/test/java/com/sysc4806app/repo/ProductRepoTest.java

```diff
@@ -0,0 +1,61 @@
+package com.sysc4806app.repo;
+
+import com.sysc4806app.model.Product;
+import com.sysc4806app.repos.ProductRepo;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
```

Comment:
Just curious. What does this do?

Reply:
it allows the @beforeAll to not be static which in turn allows the autowired repos to be used in the test.